### PR TITLE
Add Waiver new and create actions to processes waivers

### LINKED ISSUE #55: Waiver Positions Update Automatically After Waiver is Approved
When the commissioner approves a waiver, the waiver positions across the league update automatically.

### PR DESCRIPTION
**Pull request by axelclark**
Closes #55, #57, #73, #74, #75, and #76 

Adds form for admin to process waivers
When the status is updated, roster positions are created or marked "dropped", and waiver positions are updated across the league.

### CODE REVIEW

File: lib/waiver_admin.ex [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+      |> Repo.one
+
+    case waiver do
+      nil    -> nil
+      waiver -> waiver.process_at
+    end
+  end
+
+  defp three_days_from_now do
+    three_days = 86400*3
```
Comment (credoci) [inline lib/waiver_admin.ex]: There are spaces around operators most of the time, but not here.

Comment (credoci) [inline lib/waiver_admin.ex]: Large numbers should be written with underscores: 86_400

File: lib/waiver_admin.ex [hunk 2]
```diff
@@ -0,0 +1,134 @@
+defmodule Ex338.WaiverAdmin do
```
Comment (credoci) [inline lib/waiver_admin.ex]: Modules should have a @moduledoc tag.

File: lib/waiver_admin.ex (hunk 1 above)
Comment (axelclark) [inline lib/waiver_admin.ex]: Fixed

Comment (axelclark) [inline lib/waiver_admin.ex]: Fixed

File: lib/waiver_admin.ex (hunk 2 above)
Comment (axelclark) [inline lib/waiver_admin.ex]: Fixed

File: lib/waiver_admin.ex [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+      |> Repo.one
+
+    case waiver do
+      nil    -> nil
+      waiver -> waiver.process_at
+    end
+  end
+
+  defp three_days_from_now do
+    three_days = (86_400*3)
```
Comment (credoci) [inline lib/waiver_admin.ex]: There are spaces around operators most of the time, but not here.

**Pull request merged**